core/templates/components/state-editor/state-content-editor/state-content-editor.directive.html

```diff
@@ -50,19 +57,19 @@
     <button type="button" class="btn btn-secondary float-right" ng-click="cancelEdit()">Cancel</button>
     <div class="oppia-clear"></div>
   </div>
-  <div ng-if="isCardHeightLimitReached() && cardHeightLimitWarningIsShown">
-    <span class="oppia-card-height-limit-warning">
-      <i class="material-icons oppia-hide-card-height-warning-icon" uib-tooltip="Hide warning" ng-click="hideCardHeightLimitWarning()">&#xE88F;</i>
-      <span translate="I18N_CARD_HEIGHT_LIMIT_WARNING_MESSAGE"></span>
-    </span>
-  </div>
 </div>
 
 <style>
+  .material-icons {
+    color: #827127;
+  }
```

Comment:
@Rijuta-s, This is changing the color of all (At least I have seen this in 3 places) material icons in the editor pages.

Reply:
I'm going to revert this PR, @Rijuta-s can you please add a specific identifier for the color. (maybe `.oppia-card-height-limit-warning .material-icons`?)

Reply:
Expected:
![image](https://user-images.githubusercontent.com/16653571/104032635-6f883680-51f4-11eb-8dee-28f58bd989ff.png)

Obsereved:
![image](https://user-images.githubusercontent.com/16653571/104032596-60a18400-51f4-11eb-9a27-8d6d985d1fc9.png)
![image](https://user-images.githubusercontent.com/16653571/104032721-9181b900-51f4-11eb-9444-324570b9b54c.png)
![image](https://user-images.githubusercontent.com/16653571/104032740-99415d80-51f4-11eb-8b44-94271bdebf48.png)

This is happening in all the places.

Reply:
i have fixed it .
![Screenshot from 2021-01-08 21-44-24](https://user-images.githubusercontent.com/68547101/104038218-f213f480-51fa-11eb-9566-13eb269e2559.png)
Sorry for this one.

Reply:
PR to fix it : https://github.com/oppia/oppia/pull/11637